OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -25,8 +25,9 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Projectile speed in WDist / tick, two values indicate variable velocity.")]
 		public readonly WDist[] Speed = { new WDist(17) };
 
-		[Desc("Maximum offset at the maximum range.")]
-		public readonly WDist Inaccuracy = WDist.Zero;
+		[Desc("Maximum inaccuracy offset. Can have two values.",
+			"First value determinates a fixed limit, second value scales along with weapon's range.")]
+		public readonly WDist[] Inaccuracy = { WDist.Zero };
```

Comment:
I feel the description needs a bit more elaboration. Could you mention that the second value is optional, and that the first value is evaluated as well when the second is given?